.github/workflows/gpu_full_multi_gpu_tests.yml

```diff
@@ -126,30 +140,39 @@ jobs:
             echo "HOME=/home/ubuntu";
           } >> "${GITHUB_ENV}"
 
+      - name: Check device is detected
+        if: ${{ !cancelled() }}
+        run: nvidia-smi
+
       # No need to test core_crypto and classic PBS in integer since it's already tested on single GPU.
       - name: Run multi-bit CUDA integer tests
         run: |
-          make test_integer_multi_bit_gpu_ci
+          BIG_TESTS_INSTANCE=TRUE make test_integer_multi_bit_gpu_ci
```

Comment:
I grepped into the repository but couldn't deeply understand how `BIG_TESTS_INSTANCE` affects tests. I suppose this is what selects a larger selection of those, is that right?

Reply:
Yes, you can see the logic in scripts/integer-tests.sh, when the gpu feature is activated we will use:
- 8 threads for tests and doc tests when BIG_TESTS_INSTANCE is true (all H100 test workflows & the multi-GPU workflow)
- 3 threads when BIG_TESTS_INSTANCE is false (the rtx A6000 tests)

The CI passed on all devices with this configuration :slightly_smiling_face: